src/sentry_check.py

```diff
@@ -17,15 +17,13 @@ def __init__(self, filename, lines):
         self.filename = filename
         self.lines = lines
 
-        self.has_absolute_import = False
-        self.satisfies_B315_imports = False
+        # self.has_absolute_import = False
```

Comment:
Delete?

Reply:
Nah, I plan on following up using some of this to implement __future__ disallow.